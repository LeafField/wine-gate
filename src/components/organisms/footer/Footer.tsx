import React from "react";

const Footer = () => {
  return (
    <footer className="mt-45 flex h-12 items-center justify-center bg-red max-tablet:mb-12 ">
      <small className=" text-center text-xs leading-[0.75rem] text-white">
        2023Â©LeafField
      </small>
    </footer>
  );
};

export default Footer;
