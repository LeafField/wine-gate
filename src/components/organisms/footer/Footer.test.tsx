import { render } from "@testing-library/react";
import Footer from "./Footer";

describe("Footerの結合テスト", () => {
  test("スナップショット", () => {
    const { container } = render(<Footer />);
    expect(container).toMatchSnapshot();
  });
});
