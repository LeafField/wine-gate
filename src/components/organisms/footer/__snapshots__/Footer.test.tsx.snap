// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footerの結合テスト スナップショット 1`] = `
<div>
  <footer
    class="mt-45 flex h-12 items-center justify-center bg-red max-tablet:mb-12 "
  >
    <small
      class=" text-center text-xs leading-[0.75rem] text-white"
    >
      2023©LeafField
    </small>
  </footer>
</div>
`;
